app/src/main/java/com/github/multimatum_team/multimatum/model/DeadlineAdapter.kt

```diff
@@ -96,18 +98,18 @@ class DeadlineAdapter(private val context: Context) : BaseAdapter() {
                 detailTextView.setTextColor(Color.GREEN)
                 detailTextView.setTypeface(null, Typeface.BOLD)
             }
-            clockService.now() > deadline.date -> {
+            clockService.now() > deadline.dateTime -> {
                 detail = context.getString(R.string.isAlreadyDue)
             }
             else -> {
-                val timeRemaining = Period.between(clockService.now(), deadline.date)
+                val timeRemaining = clockService.now().until(deadline.dateTime, ChronoUnit.DAYS)
                 detail =
-                    context.getString(R.string.DueInXDays, timeRemaining.days.toString())
+                    context.getString(R.string.DueInXDays, timeRemaining.toString())
                 // If the remaining days is too small, put them in red or orange
-                if (timeRemaining.days < URGENT) {
+                if (timeRemaining < URGENT) {
```

Comment:
These constants (`URGENT` and `PRESSING`) should be renamed, because comparing a time and "urgent" does not make much sense conceptually. `URGENT` could e.g. become `URGENT_THRESHOLD_DAYS`. `timeRemaining` could also be renamed to `daysRemaining` (or similar) for clarity.